client/my-sites/customer-home/cards/education/educational-content/index.jsx

```diff
@@ -22,6 +22,8 @@ export default function EducationalContent( {
 	modalLinks,
 	illustration,
 	cardName,
+	width,
+	height,
```

Comment:
Adding propTypes to this might be better. It can remind us to set the width and height when adding the new one.